auth/src/authorizers/default.ts

```diff
@@ -81,7 +81,7 @@ async function allowRegisteredDID(event: APIGatewayRequestAuthorizerEvent, callb
   if (result) {
     const did = result.didResolutionResult.didDocument?.id
     const nonce = result.payload?.nonce
-    const digest = result.payload?.body
+    const digest = result.payload?.digest
```

Comment:
I think we still my have a bug, this code adds the payload but I am not seeing the `digest` in the JWS payload when I decode it manually. 

https://github.com/ceramicnetwork/js-ceramic/blob/eb179830da3b5396794e85a5df374c7fc1590829/packages/core/src/anchor/auth/did-anchor-service-auth.ts#L49-L53

Do you have any ideas on why the JWS payload coming from the Ceramic node does not have the `digest`?

Reply:
The authorizer lambda is responsible for passing this digest in a context object [here](https://github.com/ceramicnetwork/ceramic-anchor-service/blob/develop/auth/src/authorizers/default.ts#L99-L102). Then API gateway adds the header [here](https://github.com/3box/ceramic-infra/blob/dev/app-cas-auth/modules/did-auth/api_gateway.tf#L39-L41).

Reply:
What do you mean "when I decode it manually"? What are you decoding and how?

The payload from ceramic may not have a `digest` property if the request does not include a `body`.